### PR TITLE
Company names splicing fix

### PR DESCRIPTION
**Pull request by narasimhan-ramesh5**
Thanks Sue for catching the issue. I was aware of it but got sidetracked by all the other debugging.

### CODE REVIEW

File: assets/javascript/companyNames.js [hunk 1]
```diff
@@ -34,19 +34,9 @@ function getCompanyNames()
 	// We have the companies list, we gotta trim the list to 
 	// common stocks only. (type = "cs")
 	console.log(queryResponse[0].type);
-  for (i = 0; i < queryResponse.length; i++){
+  for (i = queryResponse.length-1; i > 0; i--){
 		var type = queryResponse[i].type.trim().toLowerCase();
-		var name = queryResponse[i].name.trim().toLowerCase();
```
Comment (SueJStevens) [inline assets/javascript/companyNames.js:39]: I don't think this row should be removed.

**Review by SueJStevens — CHANGES_REQUESTED** [review]
Left comment re two rows that I don't think should be removed.

File: assets/javascript/companyNames.js (hunk 1 above)
Comment (narasimhan-ramesh5) [reply assets/javascript/companyNames.js:39]: It isn't being used in the function anymore. I had it there for debugging.

Comment (narasimhan-ramesh5) [reply assets/javascript/companyNames.js:39]: I'm referring to 'var name'

**Review by SueJStevens — APPROVED** [review]
approving and merging to master.

**Pull request merged**